notebooks/.gitignore

```diff
@@ -0,0 +1,3 @@
+*
+!*.ipynb
+!.gitignore
```

Comment:
Do we really to have a .gitignore for each subdirectory?

Reply:
The main `.gitignore` on the root directory is used to ignore everything globally (i.e. what is in `klaam` directory, `scripts` directory, and other directly related directories to the logic code like the `data` directory). As `notebooks` is an external module that isn't a logic component but a different type of `scripts`, but for demoing. Here is my logic for having multiple `.gitignore`:

- `.gitignore` = global things to be ignored and what is in logic code (i.e. `klaam`).
- `notebooks/.gitignore` = ignores what is not `ipynb`
- `ckpts/.gitignore` = ignores everything but we don't GitHub to remove this folder when we push the code to GitHub servers.
- `output/.gitignore` = same thing as `ckpts/.gitignore`